### DISCUSSION TITLE
Community Version 社区版本

### DISCUSSION THREAD

**Discussion by Yue-Yang** [Yue-Yang/ChatGPT-Siri#35] [Announcements]
The following is a version where community users have modified the prompt language based on my shortcut to their local language.

# ChatGPT Siri 1.2

- [简体中文版](https://www.icloud.com/shortcuts/00838176f7f342008d5a921e8450a98d) official
- [English Version](https://www.icloud.com/shortcuts/dfa38b2abb58470380086dc4b5d50143) official
- [繁體中文版](https://www.icloud.com/shortcuts/ca0eda24dab14870affb270d286dbfbd) by @wastu01

# AI Image 1.0

- [简体中文版](https://www.icloud.com/shortcuts/b37ad418e4ec4e6d9846baa7df34e6f9) official
- [English Version](https://www.icloud.com/shortcuts/eda268ebbaa64a23bfadc53f7c13c307) official
- [繁體中文版](https://www.icloud.com/shortcuts/2a341139c95641f09331d763b965337e) by @wastu01

If you have also created a local language version, you can contact me, and after I review and confirm it, I will list it here for other users to install.

**Comment by wastu01** [comment]
推
